### DISCUSSION TITLE
Since 85, has been freezing and getting stuck, sometimes even reverting deleting text

### DISCUSSION THREAD

**Discussion by josephiesue** [athensresearch/athens#1265] [Archive]
With the change, it has been freezing and requires either to wait it out or click through and let it load and then click back and find the place you were at. Not sure what is happening, but will have to basically stop using for the time being, since it becomes more of a burden and time-slowing than actually helpful to the workflow. (as @ThatAdamGuy indicated in the comment, I am using a Mac, no lack of memory or storage, and fiber connections).

**Comment by ThatAdamGuy** [comment]
@josephiesue... curious if you're also using this on a Mac?

**Reply by josephiesue** [reply]
That i am. that I am. :/

**Reply by jefftangx** [reply]
Please update to beta.86 and let us know if any problems still exist @josephiesue @ThatAdamGuy.

**Reply by josephiesue** [reply]
Thanks @tangjeff0 it seems better at just moving through the app although still a little slow and lagging, but the biggest problem now seems to be coping a part of the list and pasting, as it will have all sorts of problems from double pasting, not pasting what is highlighted, pasting in different areas, pasting many times in multiple areas although you put it once, and when trying to delete those repeat duplicates it keeps skipping up and down the page.

**Comment by jefftangx** [comment]
I'm noticing the freezing on click too on Linux, but not sure if I understand your other problems. Is it possible to send a video recording of your issues @josephiesue @ThatAdamGuy ? X-post from #1267

**Comment by neotyk** [comment]
I've only could get cursor to disappear when arrow up key on 1st child of the page.

Doesn't froze for me, yet.

What did you do before clicking that freezes?

**Reply by jefftangx** [reply]
Demo for me: https://www.loom.com/share/4e43d6f4a5864a3ba2d31c57b4d44283

I think you can hear my clicks if sound is on

**Reply by jefftangx** [reply]
I believe this is an issue with dep upgrade from #1224 because when I checkout beta.82 and reinstall dependencies, the clicking out works again. If i only checkout without reinstalling, issue persists.

**Comment by ThatAdamGuy** [comment]
Seemed to fix stuff, thanks!
